rules/book-developers-for-a-project/rule.md

```diff
@@ -50,9 +50,8 @@ Prerequisite: [Do you install the Dynamics 365 App?](/install-the-2-add-ins)
 This is generally the easiest way for developers to book themselves in as they don't need to leave Outlook and 90% of the steps will already be familiar to them.
 
 1. Create a new Outlook appointment (in the Desktop or Web client) and give it a relevant subject
-2. Invite required resources if there are other people working with you or if you're booking for someone else  
-  **Note:** If you're booking for someone else, just have them in the To box of the appointment. If you're booking yourself AND someone else, have both of you in the To box
-
+2. Invite required resources if there are other people working with you or if you're booking for someone else\
+   **Note:** If you're booking for someone else, just have them in the To box of the appointment. If you're booking yourself AND someone else, have both of you in the To box
```

Comment:
Hey @samsmithssw 

1. Change from

> the To box

to

> the "To:" field

Reply:
Done!